vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/collect/core/model/VGSCollectFieldNameMappingPolicy.kt

```diff
@@ -5,23 +5,23 @@ package com.verygoodsecurity.vgscheckout.collect.core.model
  */
 internal enum class VGSCollectFieldNameMappingPolicy constructor(internal val analyticsName: String) {
 
+    /**
```

Comment:
@Endorf As this is an internal class migrated to checkout from collect(also there is collect in the file name), I think we can leave it as is without changes

Reply:
Ok. I'll remove `VGS Checkout supports this format by default.` because in any case, this phrase has no sens.